mlir/lib/Dialect/SPIRV/Transforms/SPIRVConversion.cpp

```diff
@@ -595,7 +595,8 @@ getOrInsertBuiltinVariable(Block &body, Location loc, spirv::BuiltIn builtin,
   case spirv::BuiltIn::WorkgroupSize:
   case spirv::BuiltIn::WorkgroupId:
   case spirv::BuiltIn::LocalInvocationId:
-  case spirv::BuiltIn::GlobalInvocationId: {
+  case spirv::BuiltIn::GlobalInvocationId:
+  case spirv::BuiltIn::GlobalOffset: {
```

Comment:
Was missing. We might want to upstream it.

Reply:
yes pls

Reply:
I added a ticket on my side to upstream this